src/components/Form.js

```diff
@@ -0,0 +1,294 @@
+import React from "react";
+import { Row, Col, Button, Container } from "react-bootstrap";
+
+class Form extends React.Component {
+  state = {
+      date: "",
+      time: "",
+      numberofguests: "",
+    fname: "Mark",
+    lname: "Otto",
+    email: "",
+    city: "",
+    state: "",
+    zip: "",
+    phonenumber: "",
+    message: ""
+  };
+
+  submitHandler = event => {
+    event.preventDefault();
+    event.target.className += " was-validated";
+  };
+
+  changeHandler = event => {
+    this.setState({ [event.target.name]: event.target.value });
+  };
+
+  render() {
+    return (
+        <Container>
+      <div className="Form">
+          <h1>BOOK A TABLE</h1>
+        <form
+          className="needs-validation"
+          onSubmit={this.submitHandler}
+          noValidate
+        >
+            <Row>
+            <Col md="4" className="mb-3">
+              <label
+                htmlFor="defaultFormRegisterNameEx"
+                className="grey-text"
+              >
+                Date
+              </label>
+              <input
+                value={this.state.date}
+                name="date"
+                onChange={this.changeHandler}
+                type="text"
+                id="defaultFormRegisterNameEx"
+                className="form-control"
+                placeholder="XX/XX/XXXX"
+                required
+              />
+              <div className="valid-feedback">Looks good!</div>
```

Comment:
I can not find any validation logic. This message appears whenever any text is added to this box even if it is not a date.